Packages/vcs/Lib/vtk_ui/textbox.py

```diff
@@ -263,25 +340,23 @@ def row_col_at_point(self, x, y):
             return row_at_point, 1
 
         # OK, no easy answer; have to calc the width of each letter in the row till we find the column
-        # Start at left or right depending on which side the click is closer to
-        if x - row_left > row_right - x:
-            # Start from right
-            for reverse_index, c in enumerate(text[::-1]):
-                w,_ = text_dimensions(c, prop)
-                if row_right - w < x:
-                    return row_at_point, len(text) - reverse_index
-                # "New" right side is one character back
-                row_right -= w
-        else:
-            # Start from left
-            for index, c in enumerate(text):
-                w, _ = text_dimensions(c, prop)
-                if row_left + w > x:
-                    return row_at_point, index
-                # "New" left side is one character forward
-                row_left += w
-        # Return the very end of the box if we can't figure it out.
-        return len(rows) - 1, len(rows[len(rows) - 1])
+        # Start from left
```

Comment:
I know this is probably a little more processor intensive, but it works, and it's silly to repeat a ton of the same code when you don't have to.